README.md

```diff
@@ -93,3 +93,11 @@ Relayed tuples are in the same order as Celestia block headers.
  **Batches**:
  The relayer informs the Blobstream contract of new data root tuple roots.
  This results in an execution of the `submitDataRootTupleRoot` function.
+
+## Audits
+
+| Date       | Auditor                                       | celestia-app                                                                        | blobstream-contracts                                                                                           | Report                                                                                                                                         |
+|------------|-----------------------------------------------|-------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2023/10/17 | [Binary Builders](https://binary.builders/)   | [v1.0.0-rc10](https://github.com/celestiaorg/celestia-app/releases/tag/v1.0.0-rc10) | [eb7a4e7](https://github.com/celestiaorg/blobstream-contracts/commit/eb7a4e74718b80277ad9dde116ead67383f5fe15) | [binary-builders.pdf](https://github.com/celestiaorg/blobstream-contracts/files/13961809/2023-10-17_Celestia_Audit_Report_Binary_Builders.pdf) |
```

Comment:
The commit hash for blobstream-contracts wasn't included in the Binary Builders PDF but Frojdi from Binary Builders shared this one.